provision.sh

```diff
@@ -45,10 +45,13 @@ apt-get install -y git bash-completion make build-essential libssl-dev \
   elixir \
   esl-erlang \
   nginx \
-  oracle-java12-installer oracle-java12-set-default maven \
-  php7.3 php7.3-mbstring \
   ponyc \
-  runit \
+  python3.8 \
```

Comment:
why install the apt version when we get it via pyenv also, later?

Reply:
Good question. I added this before I noticed the `pyenv` stuff in `user_provision.sh`. I'll remove it now.